### ISSUE TITLE
Nocturnal is damaged and can't be opened

### ISSUE THREAD

**Issue by theodorjs** [joshjon/nocturnal#8]
Thank you for developing this! I've tried installing it on my Mac mini (2014) about 10 times now, but I get the "Nocturnal is damaged and can't be opened. You should move it to the bin." message no matter what. Any ideas on how to fix this? 
_ps: this is the same comment as on Product Hunt._
![image](https://user-images.githubusercontent.com/44986321/72374121-f43b9400-3709-11ea-9fa4-3fa4f945abea.png)

**Comment by joshjon** [comment]
Hey @theodorjs! After some digging around online it seems that this may be caused by Gatekeeper (a security feature that enforces code signing and verifies downloaded applications before allowing them to run).

If you haven't already please either drag the downloaded Nocturnal.app file into /Applications manually, to implicitly consent that you're aware of this application, and allow it to run on your machine, or alternatively, right-click on its icon, choose Open, and choose Allow to make GateKeeper aware that you're explicitly trusting this application.

If that didn't work for you then you could try the solution in a similar issue raised in another GitHub repo: https://github.com/Homebrew/homebrew-cask/issues/57153#issuecomment-468817514. However, this involves using sudo to disable and re-enable Gatekeeper so don't forgot to re-enable it once you're done!

**Comment by theodorjs** [comment]
Hey @joshjon! Thank you helping me figure this out. I had already tried manually moving it to the applications folder and right-click to open, without any luck. Before I disabled GateKeeper,  I tried removing the quarantine flag from the application, using this terminal command:
_xattr -d com.apple.quarantine /path/to/program.app_

That did the trick. The dimming works perfectly, but for some reason Nigh Shift doesn't do anything. Should I open a new issue about this maybe?
